manifests/service.pp

```diff
@@ -1,6 +1,10 @@
-# == Class: fail2ban::service
+# @summary Handles the service.
+#
+# @api private
 #
 class fail2ban::service {
+  assert_private()
```

Comment:
is this maybe a breaking change when declaring this as private now?

Reply:
The README was saying it is private. So i added this. It looks used as private.
By it is a breaking change since users was able to call the class from outside before this PR.